xarray/core/dataset.py

```diff
@@ -1248,7 +1334,6 @@ def isel_points(self, dim='points', **indexers):
         Dataset.sel_points
         DataArray.isel_points
         """
```

Comment:
We want to deprecated `isel_points`, too, right?

Reply:
A warning added.